src/files.rs

```diff
@@ -139,9 +142,16 @@ fn parse_filename(filename: &str) -> MigrateResult<MigrationFile> {
     Ok(MigrationFile::new(filename, name, number, direction))
 }
 
+fn slugify(s: &str) -> String {
```

Comment:
this method does too much and is not really a slug (slug use `-`, not `_`).

How about `s.replace(" ", "-")` and letting the parse_filename handle invalid names?